src/utils/object.js

```diff
@@ -0,0 +1,15 @@
+export const isObject = obj => toString.call(obj) === '[object Object]';
```

Comment:
better to directly use `Object.prototype.toString`, now here it assumes 1. `toString` is `globalObject.toString`, which is `window.toString`, which is actually `window.__proto__.toString` (Window.prototype.toString), which is `Window.prototype.__proto__.toString` (Object.prototype.toString)